frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -231,7 +231,19 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                         } else if (source.source === SnapshotSourceType.realtime) {
                             params = { source: 'realtime' }
                         } else if (source.source === SnapshotSourceType.blob_v2) {
-                            params = { source: 'blob_v2', blob_key: source.blob_key }
+                            if (source.lts) {
```

Comment:
i wonder if we could avoid exposing this...
we're going to work on offering different retention limits and then lts stops meaning the same thing